TECHMANIA/Assets/Scripts/Components/Editor Scene/TrackSetupPanel.cs

```diff
@@ -207,7 +207,7 @@ private string CondenseFileList(List<string> fullPaths)
         string str = "";
         foreach (string file in fullPaths)
         {
-            str += Path.GetFileName(file) + "\n";
+            str += file.Remove(0, EditorContext.trackFolder.Length + 1).Replace("\\", "/") + "\n";
```

Comment:
You can make this (remove track folder and replace slash) a utility function in `Paths.cs`. And also the replacing of slashes should be specific to Windows.